data/bundle.js

```diff
@@ -54,9 +54,18 @@ window.AppData = new Proxy(defaultData, {
 //   });
 // }, 8000);
 
+function padNum(num) {
+  // input '1' => output '01'
+  return ('00' + num).substr(-2, 2);
+}
+
 function formatTime(c, d) {
   // example:   0:35 / 2:33
-  return ((c / 60).toFixed(2) + ' / ' + (d / 60).toFixed(2)).replace('.', ':');
+  const cMin = Math.floor(c / 60);
+  const cSec = Math.floor(c - cMin * 60);
+  const dMin = Math.floor(d / 60);
+  const dSec = Math.floor(d - dMin * 60);
+  return `${ cMin }:${ padNum(cSec) } / ${ dMin }:${ padNum(dSec) }`;
```

Comment:
yeah, you can ignore it for reviews. It will be ignored by git once we start hosting the addon somewhere else